workflows/fatjet_tagger.py

```diff
@@ -298,7 +298,7 @@ def process(self, events):
 
         trig_arrs = [events.HLT[_trig.strip("HLT_")] for _trig in self.triggers]
         if isRealData:
-            req_trig = np.zeros(len(events), dtype='bool')
+            req_trig = np.ones(len(events), dtype='bool')
```

Comment:
Since we have seen that the logic with which we build the trigger mask is correct, here we should restore req_trig = np.zeros(len(events), dtype='bool') and remove the 'if' statement since we apply trigger both to data and MC.